ante/sendblock.go

```diff
@@ -14,12 +14,14 @@ type SendBlockOptions struct {
 	PermittedOnlySendTo map[string]string
 }
 
```

Comment:
These files should follow godoc formating!

Reply:
Feedback implemented!